packages/superset-ui-query/src/api/v1/types.ts

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable camelcase */
+export interface V1ChartDataResponseResult {
```

Comment:
I'm not sure about the `V1` prefix. Having a prefix makes it look temporary or deprecated. Was `ChartDataResponseResult` used elsewhere?

Reply:
Agree with the doubts about variable names. This was the hardest part about this PR.
I had trouble with naming and went back and forth whether to version or not. Still not fully satisfied TBH. On one hand I want to avoid naming collision if there is v2. On the other hand the `V1xxx` looks a bit weird.

Reply:
cc @rusackas @suddjian if you have thoughts about naming.

Reply:
I think marking it as v1 is useful in cases where the interface specifically relates to behavior of the v1 api.

Reply:
I still don't feel strongly either way so I will merge and adjust later if anybody has more comments.